### DISCUSSION TITLE
Missing hemodialysis blood flow rates

### DISCUSSION THREAD

**Discussion by annabellelint** [MIT-LCP/mimic-code#1458] [MIMIC-IV]
### Prerequisites

* [X ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

I am currently trying to pull blood flow rates for ICU intermittent hemodialysis patients. I'm doing this by first identifying the hemodialysis (itemid = 225331) start and end times in procedureevents, then looking at all chartevents in that timeframe for a particular patient. However, it looks like, for many patients, blood flow rates (itemid = 224144) are only recorded during CRRT sessions, and are missing for IHD sessions. Would the blood flow rates be recorded under a different itemid for IHD patients? Or possibly in a different table? Any suggestions would be appreciated!

**Comment by alistairewj** [comment]
Not sure it's routine practice to document flow rates during IHD. When I was digging through those `itemid` I didn't see anything IHD specific. It's very likely not in the charted data in MetaVision.

**Comment by yikeshu0611** [comment]
Similarly, I am also confused, why itemid=226499 in charttevent and itemid = 225441 in procedureevents are considered to be IHD.
I hope the teacher can answer, thank you
